integration-tests/README.md

```diff
@@ -98,3 +104,29 @@ By Default test runs with those authentication parameters for Keycloak:
 ```
 
 You can update the parameters needs to configure in `integration-tests/pom.xml` or via the maven command.
+
+### Performance testing
+
+In order to be able to run the [performance tests scenarios](src/test/resources/performance-features) locally you will need to have the following installed on your machine:
+
+- [Hyperfoil v0.21](https://hyperfoil.io/) - Benchmark framework for microservices (*)
+- [webhook-perf-test](https://github.com/afalhambra/webhook-perf-test) - Dummy app to consume cloud events from webhook Processor
+
+There is a dedicated Maven profile named `performance` located in `integration-tests/pom.xml`. There you can see the following system properties needed 
+to run these performance test scenarios:
+
+```xml
+<slack.webhook.performance.url></slack.webhook.performance.url>
+<performance.hyperfoil.url></performance.hyperfoil.url>
```

Comment:
maybe we could prefix both as `performance`, so it would became `performance.slack.webhook.url`. What is your opinion?